tests/RA_Achievement_Tests.cpp

```diff
@@ -14,15 +14,15 @@ TEST_CLASS(RA_Achievement_Tests)
 public:
     TEST_METHOD(TestStateEmpty)
     {
-        Achievement ach(AchievementSetType::Local);
+        Achievement ach{ Achievement::Type::Local };
```

Comment:
Since including `RA_AchievementSet.h `was considered unnecessary, I used the Achievement version instead. Had to make this `test class `a `friend `however since it needed access.